util/logtools/generate_triggers.ts

```diff
@@ -1144,6 +1144,7 @@ Offsets: ${allOffsets.sort(numberSort).join(', ')}
       outputStrings: {
         text: {
           en: 'Custom Text',
+          fr: 'Texte personnalisé',
```

Comment:
Should these "placeholder text" be translated also ? Since they must be replaced later.

Reply:
I don't know, so I translated it anyway ;)